### DISCUSSION TITLE
Allow guests to create/edit pages?

### DISCUSSION THREAD

**Discussion by alnorris** [requarks/wiki#5505] [Help / Questions] [answered]
I'm unable to check allow guests to edit/create pages, yet they can comment. Is this feature just yet not been developed? .

**Comment by Gus19** [comment]
Hello,

The form is blocked but you can update the field in your database.

My request (pgslq) :
```update groups set permissions='["read:pages","write:pages","read:assets","read:comments","read:source","write:comments","read:history"]' where id = 2;```

After that, I update the rules, logout and it works.

**Comment by NGPixel** [comment, marked as answer]
This is not supported. Users must create an account to create/edit pages.

**Comment by pagnotta** [comment]
@NGPixel I am wondering why this is not possible. I would like to use WikiJS on a local network with a small group of people, where anybody can read/create/write/upload (and maybe edit the nav menu) without authentication (i.e. as guest). Still only the administrator would be able to make further changes.

How about an env var which enables elevating guest rights to create/write/upload content?

**Reply by NGPixel** [reply]
It's not that simple. All pages and assets require to be associated with a user ID (author / creator info). It's also used for save conflicts and other operations where multiple instances of the same guest users would introduce problems.

If it's only a small group of people, why not have them create an account and have them automatically added to a group with all necessary permissions? Once logged in, they won't need to login again as long as they use the wiki at least once every 2 weeks. You could increase that limit to something even longer under Admin > Security.
